lib/client/bst-config.ts

```diff
@@ -106,13 +106,21 @@ export class BSTConfig {
 
     private static async createConfig(nodeID?: string): Promise<any> {
         let lambdaConfig = LambdaConfig.defaultConfig().lambdaDeploy;
-        const pipeInfo = await BSTConfig.createExternalResources(nodeID);
+        try {
+            const pipeInfo = await BSTConfig.createExternalResources(nodeID);
+
+            return {
+                "sourceID": pipeInfo.endPoint.name,
+                "secretKey": pipeInfo.uuid,
+                "lambdaDeploy": lambdaConfig
+            };
+        } catch (error) {
+            if (error.code === "ETIMEDOUT") {
```

Comment:
I think this try/block should go in bst.ts (and only bst.ts - I don't think we need it in all the sub-commands, as bst is the entry point for all the calls) - it should print out the message as it is for timeouts, and for everything else it should say "Something went wrong. Please try again shortly."

It should also include the lines similar to how we handle socket issues:
https://github.com/bespoken/bst/blob/e88639bdcbe1c69501747dc69962faf8670cd5cd/lib/client/bespoke-client.ts#L54-L54

```
If the issue persists, contact us at Bespoken:
   https://gitter.im/bespoken/bst
```